Edits for Preference Center API docs (#4222)
braze-inc/braze-docs: _docs/_api/endpoints/preference_center/post_create_preference_center.md

| Parameter | Required | Data Type | Description |
| --------- | ---------| --------- | ----------- |

> |`name`| Required | String | The name of the preference center that meets the following requirements: <br>- Can only contain letters, numbers, hyphens, and underscores <br>- Cannot have spaces |

Reviewer: ```suggestion
|`name`| Required | String | The name of the preference center that meets the following requirements: <br>- Only contain letters, numbers, hyphens, and underscores <br>- Does not have spaces |
```
